src/seedu/addressbook/parser/Parser.java

```diff
@@ -85,7 +85,7 @@ public Command parseCommand(String userInput) {
             return new ClearCommand();
 
         case FindCommand.COMMAND_WORD:
-            return prepareFind(arguments);
+            return prepareFind(arguments.toLowerCase());
```

Comment:
Remember to update the documentation when you modify the behavior. (e.g. `case-sensitive` to `case-insensitive`)